src/test_l1_plat_power.c

```diff
@@ -284,22 +283,23 @@ void test_l1_plat_power_positive_PLAT_API_SetPowerState (void)
     countOfPowerState = UT_KVP_PROFILE_GET_UINT32("powermanager/CountOfPowerState");
     UT_LOG_DEBUG ("Count Of Power States %d" , countOfPowerState);
 
-    // Variation/Step 01: Call PLAT_INIT() - open interface
+    // Step 01: Call PLAT_INIT() - open interface
     ret = PLAT_INIT();
     UT_ASSERT_EQUAL_FATAL(ret, PWRMGR_SUCCESS); // Ensure the returned value is PWRMGR_SUCCESS
 
-    // Variation/step 02:Call PLAT_API_SetPowerState() -Iterate through various power states fetched from the profile file
+    // step 02:Call PLAT_API_SetPowerState() -Iterate through various power states fetched from the profile file
     for(int i = 0; i < countOfPowerState; i++) {
         snprintf(keyvalue, POWER_MANAGER_KEY_SIZE, "powermanager/PowerStates/%d", i);
         powerState = UT_KVP_PROFILE_GET_UINT32(keyvalue);
```

Comment:
Review the usage of casting:
Eg: from eIndicator = UT_KVP_PROFILE_GET_UINT32(buffer);
to eIndicator =(dsFPDIndicator_t) UT_KVP_PROFILE_GET_UINT32(buffer)

Check everywhere in document where ever you are doing GET from yaml file

Reply:
Done